locales/pt-BR/browser/browser/preferences/zen-preferences.ftl

```diff
@@ -238,7 +238,7 @@ zen-bookmark-show-library-shortcut = Mostrar Biblioteca de Favoritos
 zen-key-stop = Parar carregamento
 zen-full-zoom-reduce-shortcut = Diminuir o zoom
 zen-full-zoom-enlarge-shortcut = Aumentar zoom
-zen-full-zoom-reset-shortcut = Reset Zoom
+zen-full-zoom-reset-shortcut = Redefinir Ampliação
```

Comment:
is the idea to change only here? looks lke the word 'zoom' is used in many strings